### PR TITLE
chore: allow creation of RPC Datasource and Agent [force-release]

### PR DESCRIPTION
**Pull request by arnaud-moncel**
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: packages/datasource-toolkit/src/base-collection.ts [hunk 1]
```diff
@@ -90,15 +91,26 @@ export default abstract class BaseCollection implements Collection {
     limit?: number,
   ): Promise<AggregateResult[]>;
 
-  async execute(caller: Caller, name: string): Promise<ActionResult> {
+  async execute(
```
Comment (Thenkei) [inline packages/datasource-toolkit/src/base-collection.ts:94]: Why did you needed this change ?

Comment (arnaud-moncel) [reply packages/datasource-toolkit/src/base-collection.ts:94]: to properly override the function inside the RPC datasource
https://github.com/ForestAdmin/forestadmin-experimental/pull/89/files#diff-c5f1f283e3c2fe299c36f27cef9f2d16849b56d4413b10f9287497f873cfd043R128

File: packages/agent/src/index.ts [hunk 2]
```diff
@@ -8,8 +8,15 @@ export function createAgent<S extends TSchema = TSchema>(options: AgentOptions):
...
 
 export { Agent };
-export { AgentOptions } from './types';
+export { AgentOptions, AgentOptionsWithDefaults, HttpCode, RouteType } from './types';
 export * from '@forestadmin/datasource-customizer';
 
 // export is necessary for the agent-generator package
 export { default as SchemaGenerator } from './utils/forest-schema/generator';
+
+// export for rpc agent
```
Comment (Thenkei) [inline packages/agent/src/index.ts:17]: Maybe in an RPC object to avoid flooding the index.

**Review by Thenkei — APPROVED** [review]
Awesome. I like it this simple 👌👌

You're da best 💪

**Pull request merged**